ext/pcntl/pcntl.stub.php

```diff
@@ -1003,3 +1003,17 @@ function pcntl_setcpuaffinity(?int $process_id = null, array $cpu_ids = []): boo
 #ifdef HAVE_SCHED_GETCPU
 function pcntl_getcpu(): int {}
 #endif
+
+#ifdef HAVE_PTHREAD_SET_QOS_CLASS_SELF_NP
+enum QosClass
```

Comment:
This should be named `Pcntl\QosClass` as per https://wiki.php.net/rfc/namespaces_in_bundled_extensions.